docs/notifications.md

```diff
@@ -0,0 +1,41 @@
+# Notification Development Instructions
+
+The purpose of this document is to provide ZAP developers with simple instructions on how to add a notification.
```

Comment:
Adding an image on where notifications show up will help user understand where these notifications show up in the zap ui.

Reply:
okay